Bot/Discord.Bot.Extensions/Mafia/Models/Player.cs

```diff
@@ -20,7 +20,7 @@ public enum Team
         Blue
     }
 
-    public class Player : IUser
+    public class Player
```

Comment:
IUser is a Discord interface - here we were extending it for no reason when all we needed was encapsulation. Switching model so that we don't have to keep updating this every time they update IUser interface.